services/spar/src/Spar/Scim/User.hs

```diff
@@ -501,6 +564,20 @@ assertHandleNotUsedElsewhere hndl uid = do
   unless ((userHandle =<< musr) == Just hndl) $
     assertHandleUnused' "userName does not match UserId" hndl uid
 
+-- | Fails if the handle isn't set
```

Comment:
How can this file given that `Handle` is not a `Maybe` on the `NeededInfo` datatype?

Reply:
Because the comment is old. Thanks